tests/manage/mcg/test_namespace_crd.py

```diff
@@ -784,6 +790,148 @@ def test_create_ns_bucket_from_utilized_resources_crd(
         logger.info("Compare between uploaded files and downloaded files")
         assert self.compare_dirs(awscli_pod, amount=3)
 
+    @pytest.mark.polarion_id("OCS-2280")
+    @tier3
+    def test_create_resource_with_invalid_target_bucket_rpc(
+        self,
+        mcg_obj,
+    ):
+        """
+        Test that a proper error message is reported when invalid target
+        bucket is provided during namespace resource creation.
+        """
+        for target_bucket_name in ("", " ", "/*-#$%@^"):
+            nss_obj = NamespaceStore(
```

Comment:
I will fix it but I can not use `namespace_store_factory` as it is because it has only parameter `nss_dict` https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/namespacestore.py#L196 which accepts only dictionary of certain format. I need to create NamespaceStore with custom name and it would further increase the nss_dict complexity if there would be added another attribute to the dict (it would need to be edited in every test if it would start accepting names to the tuple with parameters that is part of the dict): https://github.com/red-hat-storage/ocs-ci/issues/4155